src/System.Xml.XDocument/tests/xNodeBuilder/CommonTests.cs

```diff
@@ -3400,7 +3401,9 @@ public void CData_4()
                 }
 
                 //[Variation(Id = 5, Desc = "WriteCData with ]]>", Priority = 1)]
-                public void CData_5()
+                [Fact]
+                [ActiveIssue(4054, default(PlatformID))]
```

Comment:
As we don't want to disable the test, I will remove the ActiveIssue attribute and as a marker, I will leave the issue id in a comment instead.